crates/next-core/src/next_config.rs

```diff
@@ -1507,31 +1487,33 @@ impl NextConfig {
     }
 
     #[turbo_tasks::function]
-    pub fn module_id_strategy_config(&self) -> Vc<OptionModuleIdStrategy> {
-        let Some(module_id_strategy) = self
-            .experimental
-            .turbo
-            .as_ref()
-            .and_then(|t| t.module_id_strategy)
-        else {
+    pub fn module_ids(&self) -> Vc<OptionModuleIds> {
+        let Some(module_ids) = self.turbopack.as_ref().and_then(|t| t.module_ids) else {
             return Vc::cell(None);
         };
-        Vc::cell(Some(module_id_strategy))
+        Vc::cell(Some(module_ids))
     }
 
     #[turbo_tasks::function]
     pub async fn turbo_minify(&self, mode: Vc<NextMode>) -> Result<Vc<bool>> {
-        let minify = self.experimental.turbo.as_ref().and_then(|t| t.minify);
-
+        let minify = self.experimental.turbopack_minify;
         Ok(Vc::cell(
             minify.unwrap_or(matches!(*mode.await?, NextMode::Build)),
         ))
     }
 
     #[turbo_tasks::function]
-    pub async fn turbo_source_maps(&self) -> Result<Vc<bool>> {
-        let source_maps = self.experimental.turbo.as_ref().and_then(|t| t.source_maps);
+    pub async fn client_source_maps(&self, mode: Vc<NextMode>) -> Result<Vc<bool>> {
+        let source_maps = self.experimental.turbopack_source_maps;
+        Ok(Vc::cell(source_maps.unwrap_or(match &*mode.await? {
+            NextMode::Build => self.production_browser_source_maps,
+            NextMode::Development => true,
+        })))
+    }
```

Comment:
@timneutkens `experimental.turbopackSourceMaps` takes precedence over `productionBrowserSourceMaps`, which should be good if the main use of `turbopackSourceMaps` is debugging?